pkg/containers/containers.go

```diff
@@ -90,6 +90,47 @@ func (c *Containers) procMountsCgroups() error {
 	return nil
 }
 
+//returns the ContainerID of a given mountpoint cgourp
+func getContainerIdFromTaskContainer(taskContainer *Containers, cgroup string) string {
+	taskContainer.procMountsCgroups()
+	path := fmt.Sprintf("%s%s", taskContainer.cgroupMP, cgroup)
+	containerId, _ := getContainerIdFromCgroup(path)
+	return containerId
+}
+
+// gets a containerID from a given task or process directory path
+func GetContainerIdFromTaskDir(taskPath string) (string, error) {
+	containerId := ""
+	taskPath = fmt.Sprintf("%s/cgroup", taskPath)
+	cgroupFile, err := os.Open(taskPath)
+	if err != nil {
+		return containerId, err
+	}
+	defer cgroupFile.Close()
+	scanner := bufio.NewScanner(cgroupFile)
+	taskContainer := InitContainers()
```

Comment:
If you will make this a function of Containers struct, you won't have to initialize it again (calling it from the instance held by tracee).
On the other hand, that might not be neccessary and you can probably remove this line anyways - see my next comment